src/earn/types.ts

```diff
@@ -69,11 +67,11 @@ export interface BeforeDepositAction {
 }
 
 export interface WithdrawAction {
-  name: WithdrawActionName
+  name: EarnActiveAction
```

Comment:
I don't think we should be using a broader type here. Since `deposit`, `swap-deposit` aren't valid "withdraw" actions. This opens up the possibility of bugs / handling invalid types. Can we still keep a narrow type here? You can still have a broader type if it makes things easier in other places, but we should keep types as narrow as possible

Reply:
Narrowed in https://github.com/valora-inc/wallet/commit/ff2bd43cf3f9794d2b053f71d3ea04e459f2698f!